arbiter-core/src/middleware/cast.rs

```diff
@@ -1,4 +1,4 @@
-///! Utility functions for casting between revm and ethers-rs types.
+/// ! Utility functions for casting between revm and ethers-rs types.
```

Comment:
I don't think this comment makes sense and I'm not sure what the `!` is for

Reply:
I didn't write this comment, but it looks like `cargo fmt` hit it. I think it makes sense as a module comment doesn't it?